### PR TITLE
add blog

### PR DESCRIPTION
**Pull request by PacoGD**

### CODE REVIEW

File: staff/PacoGD/blog/links/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (MuyZan) [inline staff/PacoGD/blog/links/index.html:1]: **staff/PacoGD/blog/links/index.html!!!**

Watch out!! better use another name for a non principal html file.

File: staff/PacoGD/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                quo sit illum facere fugit corrupti. Nam eligendi a animi veniam nemo repudiandae quae nisi tempore ipsa!</p>
+            <footer>
+                Nacho Soto
+            </footer>
+        </article>
+        <nav>  <a href="https://www.muyinteresante.es/salud/fotos/alimentos-con-muchas-vitaminas" target="_blank">
+            <h2>Links páginas</h2>
+        </a>
+            <!-- Es donde pondremos los links relacionados con el tema de navegación, en un blog por ejemplo serian los últimos artículos relacionados con el tema-->
+        </nav>
```
Comment (MuyZan) [inline staff/PacoGD/blog/index.html:58]: Usually this kind of link section is inside an < aside > tag.

File: staff/PacoGD/blog/links/index.html (hunk 1 above)
Comment (PacoGD) [reply staff/PacoGD/blog/links/index.html:1]: Done!

File: staff/PacoGD/blog/index.html (hunk 2 above)
Comment (PacoGD) [reply staff/PacoGD/blog/index.html:58]: Done!

**Pull request merged**